### ISSUE TITLE
Possible implementation on ROS2 Humble

### ISSUE THREAD

**Issue by Astik-2002** [USTC-AIS-Lab/PE-Planner#1]
Greetings. I was wondering if PE planner can be implemented using ros2 humble instead of noetic. I'm working on a similar project related to agile drone flight, and my lower level control stack is built on ros2, so I was hoping to get PE planner working with it.

**Comment by kuguao** [comment]
Of course, PE-Planner can be implemented on ROS2. However, we have no plans for this at the moment. We suggest modifying the code in "modules/px4_interface" and "/modules/ros_interface" to adapt it to ROS2 and your lower level controller.
